tools/check_contrast.py

```
@@ -0,0 +1,160 @@
+
+    def effective(name, mode):
+        if name == "(default)":
+            return dict(root_light) if mode == "light" else {**root_light, **root_dark}
+        t = themes[name]
+        if mode == "light":
+            return {**root_light, **t["light"]}
+        return {**root_light, **root_dark, **t["light"], **t["dark"]}
+
+    fails = []
+    checked = 0
+    for name in ["(default)"] + sorted(themes):
```

Ruff (0.16.1)

[warning] Consider `["(default)", *sorted(themes)]` instead of concatenation

Replace with `["(default)", *sorted(themes)]`

(RUF005)